webgpu.h

```diff
@@ -901,6 +923,15 @@ WGPU_EXPORT void wgpuRenderPassEncoderSetViewport(WGPURenderPassEncoder renderPa
 WGPU_EXPORT WGPUTextureView wgpuTextureCreateView(WGPUTexture texture, WGPUTextureViewDescriptor const * descriptor);
 WGPU_EXPORT void wgpuTextureDestroy(WGPUTexture texture);
 
+// Methods of SwapChain
+WGPU_EXPORT WGPUTextureView wgpuSwapChainGetCurrentTextureView(WGPUSwapChain swapChain);
+WGPU_EXPORT void wgpuSwapChainPresent(WGPUSwapChain swapChain);
+
+// Surface creation
```

Comment:
What do you think of using descriptors for these too for extensibility? Also I'm thinking this might need an "Instance" argument somehow, which isn't something we have in this header for now.

Reply:
That's pretty cool, although I'm not sure what to do when a new platforms comes by (Fuchsia?). Would it be an extension on the descriptor?

Alternatively we could use the sType argument like Vulkan to be able to pass several differently shaped structures.

(also there are not proc definitions for these functions).

Reply:
IMO, an sType would be pretty nice here.

Reply:
Looks good, last comment is that the WGPUProcCreateSurface is missing. (and we can take care of the instance argument separately, when we resolve whether to have an instance or not)

Reply:
I wasn't sure if instance-level entry points need a proc in your implementation. It's not like you can query those addresses anyway.